<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MY COMPASS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="goalLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>751</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Main goal is not set.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addGoalButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Goal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editGoalButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Goal</string>
    </property>
   </widget>
   <widget class="QLabel" name="milestoneLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Key mid-term milestones:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="milestoneListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>751</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addMilestoneButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>150</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Milestone</string>
    </property>
   </widget>
   <widget class="QLabel" name="taskLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Your current tasks:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Task description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dueDateEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>YYYY-MM-DD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="priorityComboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addTaskButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>150</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Task</string>
    </property>
   </widget>
   <widget class="QListWidget" name="taskListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>751</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
